tracer/src/Datadog.Trace/Tagging/AwsSnsTags.cs

```diff
@@ -3,6 +3,8 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#nullable enable
```

Comment:
Makes me think I should update the source generator to output nullable enabled code too 🤔